pkg/proxy/ipvs/proxier.go

```diff
@@ -1637,6 +1648,17 @@ func (proxier *Proxier) writeIptablesRules() {
 		"-j", "DROP",
 	)
 
+	// disable LB access from node
+	// for IPVS src and dst both would be lbIP
+	for _, lbIP := range proxier.lbNoNodeAccessIPs.UnsortedList() {
+		proxier.filterRules.Write(
+			"-A", string(utiliptables.ChainOutput),
+			"-s", lbIP,
+			"-m", "ipvs", "--vaddr", lbIP,
+			"-j", "DROP",
+		)
```

Comment:
Tried this locally on kind, every iptables-restore call is creating a duplicate entry in iptable rules. 

After few minutes I ended up with 18 exact same rules 😔 

```
╰─$ docker exec -it kind-worker /bin/bash                                                                                                                                                                       130 ↵
root@kind-worker:/# iptables-save | grep ipvs 
-A INPUT -m comment --comment "kubernetes ipvs access filter" -j KUBE-IPVS-FILTER
-A OUTPUT -s 172.18.255.200/32 -m ipvs --vaddr 172.18.255.200 -j DROP
-A OUTPUT -s 172.18.255.200/32 -m ipvs --vaddr 172.18.255.200 -j DROP
-A OUTPUT -s 172.18.255.200/32 -m ipvs --vaddr 172.18.255.200 -j DROP
-A OUTPUT -s 172.18.255.200/32 -m ipvs --vaddr 172.18.255.200 -j DROP
-A OUTPUT -s 172.18.255.200/32 -m ipvs --vaddr 172.18.255.200 -j DROP
-A OUTPUT -s 172.18.255.200/32 -m ipvs --vaddr 172.18.255.200 -j DROP
-A OUTPUT -s 172.18.255.200/32 -m ipvs --vaddr 172.18.255.200 -j DROP
-A OUTPUT -s 172.18.255.200/32 -m ipvs --vaddr 172.18.255.200 -j DROP
-A OUTPUT -s 172.18.255.200/32 -m ipvs --vaddr 172.18.255.200 -j DROP
-A OUTPUT -s 172.18.255.200/32 -m ipvs --vaddr 172.18.255.200 -j DROP
-A OUTPUT -s 172.18.255.200/32 -m ipvs --vaddr 172.18.255.200 -j DROP
-A OUTPUT -s 172.18.255.200/32 -m ipvs --vaddr 172.18.255.200 -j DROP
-A OUTPUT -s 172.18.255.200/32 -m ipvs --vaddr 172.18.255.200 -j DROP
-A OUTPUT -s 172.18.255.200/32 -m ipvs --vaddr 172.18.255.200 -j DROP
-A OUTPUT -s 172.18.255.200/32 -m ipvs --vaddr 172.18.255.200 -j DROP
-A OUTPUT -s 172.18.255.200/32 -m ipvs --vaddr 172.18.255.200 -j DROP
-A OUTPUT -s 172.18.255.200/32 -m ipvs --vaddr 172.18.255.200 -j DROP
-A OUTPUT -s 172.18.255.200/32 -m ipvs --vaddr 172.18.255.200 -j DROP
root@kind-worker:/# 
```

Also, checked the `proxier.filterRules` it only has single entry for this rule. I guess duplication has something to do with the flags which we pass to restore call.